### ISSUE TITLE
Cannot save notebook on VPN when pyviz/hvplot or bokeh have outputs

### ISSUE THREAD

**Issue by ericmjl** [pyviz/hvplot#260]
I understand that this, at first glance, doesn't sound like a pyviz/hvplot or bokeh issue, but I ask that you hear me out and watch the screencast before judging.

The issue I'm facing is that when I have hvplot or bokeh outputs in my notebook, I am suddenly unable to save my notebook when connected to my company VPN. I have gone back-and-forth with our networking team, and when I finally narrowed down the issue, it turned out the minimal reproducible example was the one in the screencast.

This notebook cell would have no issues:

```python
%%capture
import hvplot.pandas
import holoviews as hv

hv.extension("bokeh")
```

While this notebook cell would cause issues with saving:

```python
import hvplot.pandas
import holoviews as hv

hv.extension("bokeh")
```

The only difference is whether or not I capture the output.

#### ALL software version info (hvPlot, holoviews, bokeh, python, notebook, OS, browser, any other relevant packages)

hvplot: 0.4.0
bokeh: 1.3.0
python: 3.7
jupyterlab: 1.0.2
jupyterlab_server: 1.0.0
jupyter: 1.0.0

Browser: Chrome 75.0.3770.142
OS: macOS 10.14.5

#### Stack traceback and/or browser JavaScript console output

Please see screencast for this.

#### Screenshots or screencasts of the bug in action

Screencast: https://drive.google.com/file/d/1ZI8AErWtU-7ZdwyranCOh205ykCgFCha/view?usp=sharing

**Comment by philippjfr** [comment]
@ericmjl Do you know if this is still an issue?

**Comment by ericmjl** [comment]
@philippjfr yes, it has been an issue for a while, and I'm not quite sure how to solve it. Thanks for pinging back, btw. :)

**Comment by m3rickson** [comment]
I also have this issue, but the minimum reproducible example is simply

`import hvplot.xarray`

**Comment by maximlt** [comment]
That is a pretty scary bug! If any of you is still using hvPlot and encounter this issue, please let us know so that we investigate further.
